client/utils/events.ts

```diff
@@ -80,6 +80,13 @@ function isEventSameDay(startingDate: IDateTime, endingDate: IDateTime): boolean
     return moment(startingDate).format('DD/MM/YYYY') === moment(endingDate).format('DD/MM/YYYY');
 }
 
+function showEventStartDate(eventDate: IDateTime, planningDate?: IDateTime): boolean {
+    if (planningDate == null) {
+        return true;
+    }
+    return moment(eventDate).format('DD/MM/YYYY') != moment(planningDate).format('DD/MM/YYYY');
```

Comment:
it'd be cleaner to use `isSame` method from moment. We don't care about date format here